imagepipeline/src/main/java/com/facebook/imagepipeline/core/ImagePipeline.java

```diff
@@ -391,6 +391,28 @@ public boolean isInBitmapMemoryCache(final ImageRequest imageRequest) {
 
   /**
    * Returns whether the image is stored in the disk cache.
+   * The check is made synchronously
```

Comment:
This doesn't look complete. The Javadoc of `isInDiskCacheSync(ImageRequest)` is more explicit about blocking the thread so please add the same or a similar clarification here.